src/components/footer/footer.scss

```diff
@@ -15,21 +15,24 @@
   }
 }
 
+.App-footer {
+  background-color: #00478f;
+}
+
 .footer-content {

```

Comment:
I know it wasn't part of this task, but just found out that we have this issue when having more than one column in the footer. Do you mind checking it? Perhaps having the storybook example updated with 2 columns as well? 
![Screenshot 2020-03-19 at 00 17 38](https://user-images.githubusercontent.com/1295800/77016765-f247b880-6978-11ea-9524-eebab3b04397.png)